### ISSUE TITLE
Account blocking

### ISSUE THREAD

**Issue by Egor-Varaksa** [multitheftauto/mtasa-blue#1998]
Hello, please add the ability to ban by login. Let's say I ban the login 'Enot' and when the player enters the password under this username , he will write that the account is blocked and will not allow you to log in to the account. This will be very convenient if the player has multiple accounts and I need to block the account instead of serial or ip. And this is how you can do it in the banlist: <ban login="Enot" account="Enot" banner="Console" reason="reason" time="1609246634"></ban>
 
![image](https://user-images.githubusercontent.com/77339001/104329246-c72ce780-54fd-11eb-865b-435f65e2d7c8.png)

**Comment by Egor-Varaksa** [comment]
<ban login="Enot" account="Enot" banner="Console" reason="reason" time="1609246634"></ban>

**Comment by uw935** [comment]
Yes, that would be cool.

**Comment by xLive** [comment]
You can easily do it in Lua
[onPlayerLogin](https://wiki.multitheftauto.com/wiki/OnPlayerLogin)

**Comment by Egor-Varaksa** [comment]
> Вы можете легко сделать это в Lua
> [onPlayerLogin](https://wiki.multitheftauto.com/wiki/OnPlayerLogin)

"Yes, I know. But it is much more convenient that all this is stored in the banlist and can be managed through the admin panel

**Comment by Egor-Varaksa** [comment]
> Вы можете легко сделать это в Lua
> [onPlayerLogin](https://wiki.multitheftauto.com/wiki/OnPlayerLogin)

Yes, I know. But it is much more convenient that all this is stored in the banlist and can be managed through the admin panel

**Comment by LosFaul** [comment]
i think:
banPlayer should be modified to allow banning accounts without passing player element
or a new function banAccount should be added

**Comment by patrikjuvonen** [comment]
Wondering if we could refactor the unused "username" field into account name?

**Comment by patrikjuvonen** [comment]
ccw confirmed that refurbishing the username/account field will not affect MTA backend. So, good to go :+1:
